src/GameLogic/DataStructures/ActionSequence.ts

```diff
@@ -0,0 +1,24 @@
+import {PlayerAction} from "../Enums";
```

Comment:
Some weird stuff happened with whitespace here but don't bother with it - I'll install eslint into the project and add a precommit hook to lint and fix, this should fix any whitespace and style woes we might have.